staging/src/k8s.io/apiserver/pkg/admission/cel/metrics.go

```diff
@@ -98,8 +98,14 @@ func (m *ValidatingAdmissionPolicyMetrics) ObserveDefinition(ctx context.Context
 	m.policyDefinition.WithContext(ctx).WithLabelValues(state, enforcementAction).Inc()
 }
 
-// ObserveCheck observes a policy validation check.
-func (m *ValidatingAdmissionPolicyMetrics) ObserveCheck(ctx context.Context, elapsed time.Duration, policy, binding, expression, enforcementAction, params, state string) {
-	m.policyCheck.WithContext(ctx).WithLabelValues(policy, binding, expression, enforcementAction, params, state).Inc()
-	m.policyLatency.WithContext(ctx).WithLabelValues(policy, binding, expression, enforcementAction, params, state).Observe(elapsed.Seconds())
+// ObserveAdmissionWithError observes a policy validation error that was ignored due to failure policy.
```

Comment:
I thought we had four possibilities:
```
allow with ignored error
allow
deny with error
deny
```

it's not clear (to me) if `ObserveRejection observes a policy validation error.` means an evaluation error occurred, or the evaluation succeeded and decided to reject the request

Reply:
Gotcha, I'll change it to `ObserveRejection observes a policy validation error that was at least one of the reasons for a deny` - I think "one of" makes sense here, since multiple `ObserveRejectionWithError` observations can occur on a single policy.

Reply:
Could we have ObserveDeny, ObserveDenyWithError, ObserveAdmit, ObserveAdmitWithError? We could add some in a follow-up as long as current metric states clear